src/pages/Entries/HappinessCard.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import Card from "../../components/Card";
+import { Happiness } from "../../data/models/Happiness";
+
+export default function HappinessCard({
```

Comment:
Add documentation for this component, it's hard to tell what it's used for just from the name.